rma_account/models/account_move_line.py

```diff
@@ -21,7 +21,11 @@ def _name_search(self, name, domain=None, operator="ilike", limit=None, order=No
                     [("display_type", "in", ("product", "line_section", "line_note"))],
                 ]
             )
-        lines = self.search([("move_id.name", operator, name)] + domain, limit=limit)
+        lines = self.search(
+            ["|", ("move_id.name", operator, name), ("move_id.ref", operator, name)]
```

Comment:
@Nikul-OSI This field will not have sale order number, there is a another field called invoice_origin check for that if that works. Or find another solutin.

Reply:
@Chanakya-OSI Yes Code updated with **invoice_origin**. Now Please take a look.

![image](https://github.com/ursais/stock-rma/assets/98144048/49d41c7b-3162-4ead-9258-8a6f32c375e5)